packages/credit-card-integration/src/index.ts

```diff
@@ -0,0 +1,3 @@
+// export { default as createCreditCardPaymentStrategy} from "./lib/create-credit-card-payment-strategy";
```

Comment:
do we need this comment?

Reply:
I removed `create-credit-card-payment-strategy` from this PR. 
Once we've decided whether or not to include it, I'll remove or uncomment this line.